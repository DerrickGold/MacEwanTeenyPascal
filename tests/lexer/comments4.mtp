(* regular comment
that spans
a bunch of
lines (*some other comment too

should return an end file token!
*)