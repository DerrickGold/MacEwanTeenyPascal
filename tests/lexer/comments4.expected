7, TOK_ENDFILE,
